.circleci/config.yml

```diff
@@ -141,37 +116,24 @@ jobs:
     <<: *macos_defaults
     steps:
       - *addWorkspace
-      - restore-cache: *cache_restore_brew
```

Comment:
Had to remove the cache due to them being so stale, they wouldn't run on a more recent VM.